apps/srt-file-transmit.cpp

```diff
@@ -180,7 +180,7 @@ int parse_args(FileTransmitConfig &cfg, int argc, char** argv)
         return 2;
     }
 
-    cfg.chunk_size    = stoul(Option<OutString>(params, "1316", o_chunk));
+    cfg.chunk_size    = stoul(Option<OutString>(params, "1456", o_chunk));
```

Comment:
Actually I doubt this option makes any sense anymore.

Reply:
I agree it is very unlikely to be used, but still is was in the previous file transmit version, so I'd better keep it, at least to minimize the work on these apps.